Import failure on 32-bit systems

I'm on master for CUDAdrv, CUDAnative, CUDAapi, CuArrays & Flux, and see this on Travis for this setup:
https://travis-ci.com/ianshmean/YOLO.jl/jobs/259310409
```
Julia Version 1.3.0-rc5.1
Platform Info:
  OS: Linux (i686-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU @ 2.30GHz
  WORD_SIZE: 32
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  TRAVIS_JULIA_VERSION = 1.3
  JULIA_NUM_THREADS = 2
  JULIA_PROJECT = @.
```

```
ERROR: LoadError: LoadError: AssertionError: sizeof(CUdeviceptr) == sizeof(Culonglong)
Stacktrace:
 [1] top-level scope at /home/travis/.julia/packages/CUDAdrv/fKlYl/src/libcuda_common.jl:45
 [2] include at ./boot.jl:328 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1105
 [4] include at ./Base.jl:31 [inlined]
 [5] include(::String) at /home/travis/.julia/packages/CUDAdrv/fKlYl/src/CUDAdrv.jl:1
 [6] top-level scope at /home/travis/.julia/packages/CUDAdrv/fKlYl/src/CUDAdrv.jl:18
 [7] include at ./boot.jl:328 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1105
 [9] include(::Module, ::String) at ./Base.jl:31
 [10] top-level scope at none:2
 [11] eval at ./boot.jl:330 [inlined]
 [12] eval(::Expr) at ./client.jl:425
 [13] top-level scope at ./none:3
in expression starting at /home/travis/.julia/packages/CUDAdrv/fKlYl/src/libcuda_common.jl:45
in expression starting at /home/travis/.julia/packages/CUDAdrv/fKlYl/src/CUDAdrv.jl:18
ERROR: LoadError: Failed to precompile CUDAdrv [c5f51814-7f29-56b8-a69c-e4d8f6be1fde] to /home/travis/.julia/compiled/v1.3/CUDAdrv/HMhfu_5fUXO.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1283
 [3] _require(::Base.PkgId) at ./loading.jl:1024
 [4] require(::Base.PkgId) at ./loading.jl:922
 [5] require(::Module, ::Symbol) at ./loading.jl:917
 [6] include at ./boot.jl:328 [inlined]
 [7] include_relative(::Module, ::String) at ./loading.jl:1105
 [8] include(::Module, ::String) at ./Base.jl:31
 [9] top-level scope at none:2
 [10] eval at ./boot.jl:330 [inlined]
 [11] eval(::Expr) at ./client.jl:425
 [12] top-level scope at ./none:3
in expression starting at /home/travis/.julia/packages/CUDAnative/Qi9dJ/src/CUDAnative.jl:4
ERROR: LoadError: Failed to precompile CUDAnative [be33ccc6-a3ff-5ff2-a52e-74243cff1e17] to /home/travis/.julia/compiled/v1.3/CUDAnative/4Zu2W_5fUXO.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1283
 [3] _require(::Base.PkgId) at ./loading.jl:1024
 [4] require(::Base.PkgId) at ./loading.jl:922
 [5] require(::Module, ::Symbol) at ./loading.jl:917
 [6] include at ./boot.jl:328 [inlined]
 [7] include_relative(::Module, ::String) at ./loading.jl:1105
 [8] include(::Module, ::String) at ./Base.jl:31
 [9] top-level scope at none:2
 [10] eval at ./boot.jl:330 [inlined]
 [11] eval(::Expr) at ./client.jl:425
 [12] top-level scope at ./none:3
```

--- a/res/patches/cudeviceptr.patch
+++ b/res/patches/cudeviceptr.patch
@@ -5,7 +5,6 @@
  const cuuint32_t = UInt32
  const cuuint64_t = UInt64
 -const CUdeviceptr = Culonglong
-+@assert sizeof(CUdeviceptr) == sizeof(Culonglong)
  const CUdevice = Cint
  const CUctx_st = Cvoid
  const CUcontext = Ptr{CUctx_st}

--- a/src/libcuda_common.jl
+++ b/src/libcuda_common.jl
@@ -42,7 +42,6 @@ const CU_PARAM_TR_DEFAULT = -1
 
 const cuuint32_t = UInt32
 const cuuint64_t = UInt64
-@assert sizeof(CUdeviceptr) == sizeof(Culonglong)
 const CUdevice = Cint
 const CUctx_st = Cvoid
 const CUcontext = Ptr{CUctx_st}

